client/SFTPClientGUI/SFTPClientGUI.py

```diff
@@ -0,0 +1,366 @@
+		col = Gtk.TreeViewColumn('Files')
+		self.logger = logger
+		self.treeview_local = treeview
+		col_text = Gtk.CellRendererText()
+		col_img = Gtk.CellRendererPixbuf()
+		col.pack_start(col_img, False)
+		col.pack_start(col_text, True)
+		col.add_attribute(col_text, 'text', 0)
+		col.add_attribute(col_img, 'pixbuf', 1)
+		col.set_sort_column_id(0)
+		col_perm = Gtk.TreeViewColumn('Permissions')
+		col_perm.pack_start(col_text, True)
+		col_perm.add_attribute(col_text, 'text', 3)
+		col_size = Gtk.TreeViewColumn('Size')
+		col_size.pack_start(col_text, True)
+		col_size.add_attribute(col_text, 'text', 4)
+		col_size.set_sort_column_id(5)
+		col_date = Gtk.TreeViewColumn('Date Modified')
+		col_date.pack_start(col_text, True)
+		col_date.add_attribute(col_text, 'text', 6)
+		col_date.set_sort_column_id(6)
+		self.treeview_local.append_column(col)
+		self.treeview_local.append_column(col_perm)
+		self.treeview_local.append_column(col_size)
+		self.treeview_local.append_column(col_date)
+
+		self.treeview_local.connect('row-expanded', self.signal_expand_row)
+		self.treeview_local.connect('row-collapsed', self.signal_collapse_row)
+		self._tvmodel = Gtk.TreeStore(str, Pixbuf, str, str, str, int, str)
+
+		self.local_hidden = True
+		self._get_popup_menu()
+
+	def _get_popup_menu(self):
+		self.treeview_local.connect('button_press_event', self._signal_treeview_button_pressed)
+
+		self.popup_menu = Gtk.Menu.new()
+		menu_item = Gtk.CheckMenuItem.new_with_label('Show Hidden Files')
+		menu_item.connect('toggled', self.signal_menu_toggled_hidden_files)
+		self.signal_menu_toggled_hidden_files = menu_item
+		self.popup_menu.append(menu_item)
+
+		menu_item = Gtk.MenuItem.new_with_label('Delete')
+		menu_item.connect('activate', self.signal_menu_activate_delete_prompt)
+		self.popup_menu.append(menu_item)
+		self.popup_menu.show_all()
+
+	def _signal_treeview_button_pressed(self, _, event):
+		if event.button == Gdk.BUTTON_SECONDARY:
+			self.popup_menu.popup(None, None, None, None, event.button, Gtk.get_current_event_time())
+			return True
+		return
+
+	def signal_expand_row(self, _, treeiter, treepath):
+		new_path = self._tvmodel[treeiter][2]  # pylint: disable=unsubscriptable-object
+		self.load_dirs(new_path, treeiter)
+		self._tvmodel.remove(self._tvmodel.iter_children(treeiter))
+
+	def signal_collapse_row(self, _, treeiter, treepath):
+		current = self._tvmodel.iter_children(treeiter)
+		while current:
+			self._tvmodel.remove(current)
+			current = self._tvmodel.iter_children(treeiter)
+		self._tvmodel.append(treeiter, [None, None, None, None, None, None, None])
+
+	def load_dirs(self, path, parent=None):
+		counter = 0
+		for name in self._yield_dir_list(path):
+			if self.local_hidden and name.startswith('.'):
+				continue
+			fullname = path + '/' + name
+			perm = self._check_perm(fullname)
+			raw_time = self._get_raw_time(fullname)
+			date_modified = '   ' + time.strftime('%y-%m-%d %H:%M:%S', time.localtime(raw_time))
```

Comment:
This should instead use the [utilities module](https://github.com/securestate/king-phisher/blob/master/king_phisher/utilities.py#L252), you'll have to convert the time to an appropriate object first.